Enable pydocstyle pre-commit hook

VannTen opened the pull request:
WIP. Fix commits welcome

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **pacospace** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @pacospace` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=thoth-station%2Fstorages).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/thoth-station/storages/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["pacospace"]} -->

sesheta commented:
@VannTen: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
aicoe-ci/prow/pre-commit | be42b6b546660bde34cd879dfcf7494d1d5be16e | [link](https://prow.operate-first.cloud/view/s3/ci-prow/prow-logs/pr-logs/pull/thoth-station_storages/2660/pre-commit/1547195440871510016) | true | `/test pre-commit`

[Full PR test history](https://prow.operate-first.cloud/pr-history?org=thoth-station&repo=storages&pr=2660). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3AVannTen). Please help us and [open an issue](https://github.com/thoth-station/storages/issues?q=is:issue+is:open) when you hit one in your PR.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

VannTen commented:
/kind cleanup

sesheta commented:
@VannTen: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
aicoe-ci/prow/pre-commit | d8d612617fed37d6b7650c5cf5b6ca7887f24c86 | [link](https://prow.operate-first.cloud/view/s3/ci-prow/prow-logs/pr-logs/pull/thoth-station_storages/2660/pre-commit/1547208921440063488) | true | `/test pre-commit`

[Full PR test history](https://prow.operate-first.cloud/pr-history?org=thoth-station&repo=storages&pr=2660). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3AVannTen). Please help us and [open an issue](https://github.com/thoth-station/storages/issues?q=is:issue+is:open) when you hit one in your PR.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **sesheta** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @sesheta` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=thoth-station%2Fstorages).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/thoth-station/storages/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["sesheta"]} -->

sesheta commented:
@VannTen: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
aicoe-ci/prow/pre-commit | c9ca542d6c35a705433f1ff7b1f249a9b0515927 | [link](https://prow.operate-first.cloud/view/s3/ci-prow/prow-logs/pr-logs/pull/thoth-station_storages/2660/pre-commit/1551575687301697536) | true | `/test pre-commit`

[Full PR test history](https://prow.operate-first.cloud/pr-history?org=thoth-station&repo=storages&pr=2660). [Your PR dashboard](https://prow.operate-first.cloud/pr?query=is%3Apr%20state%3Aopen%20author%3AVannTen). Please help us and [open an issue](https://github.com/thoth-station/storages/issues?q=is:issue+is:open) when you hit one in your PR.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

VannTen commented:
Ready for review :+1:

mayaCostantini commented on setup.py:
```diff
@@ -6,19 +8,18 @@
-    with open("requirements-test.txt", "r") as requirements_file:
-        res = requirements_file.readlines()
-        return [req.split(" ", maxsplit=1)[0] for req in res if req]
+get_test_requires = get_install_requires
```
Don't we need to get `requirements-test.txt` here instead of `requirements.txt`?

mayaCostantini commented on setup.py:
```diff
@@ -1,3 +1,5 @@
+"""Setup configuration for adviser module."""
```
```suggestion
"""Setup configuration for storages module."""
```

mayaCostantini commented on setup.py:
```diff
@@ -6,19 +8,18 @@
 
 
 def get_install_requires():
+    """Get requirements for adviser module."""
```
```suggestion
    """Get requirements for storages module."""
```

mayaCostantini commented on setup.py:
```diff
@@ -6,19 +8,18 @@
 
 
 def get_version():
+    """Get current version of adviser module."""
```
```suggestion
    """Get current version of storages module."""
```

mayaCostantini reviewed (commented):
lgtm, just a few details :+1:

VannTen commented:
On Tue, Jul 26, 2022 at 03:25:44AM -0700, Maya Costantini wrote:
> @mayaCostantini commented on this pull request.
> 
> lgtm, just a few details :+1: 
> 
> >      with open("requirements.txt", "r") as requirements_file:
>          # TODO: respect hashes in requirements.txt file
>          res = requirements_file.readlines()
>          return [req.split(" ", maxsplit=1)[0] for req in res if req]
>  
>  
> -def get_test_requires():
> -    with open("requirements-test.txt", "r") as requirements_file:
> -        res = requirements_file.readlines()
> -        return [req.split(" ", maxsplit=1)[0] for req in res if req]
> +get_test_requires = get_install_requires
> 
> Don't we need to get `requirements-test.txt` here instead of `requirements.txt`?


Oh that's embarrassing. Code-climate was yelling that the two code
blocks were identical and my eyes could not make the diff.

> 
> > @@ -1,3 +1,5 @@
> +"""Setup configuration for adviser module."""
> 
> ```suggestion
> """Setup configuration for storages module."""
> ```
> 
> > @@ -6,19 +8,18 @@
>  
>  
>  def get_install_requires():
> +    """Get requirements for adviser module."""
> 
> ```suggestion
>     """Get requirements for storages module."""
> ```
> 
> >  
>  
>  def get_version():
> +    """Get current version of adviser module."""
> 
> ```suggestion
>     """Get current version of storages module."""
> ```
> 

ACK, I'll fix all of that.

mayaCostantini commented:
Thanks @VannTen :smile:

harshad16 reviewed (approved):
/lgtm

one note would be squash the commits to 1 . 
thanks :100:

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/thoth-station/storages/pull/2660#pullrequestreview-1054606113" title="LGTM">harshad16</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **sesheta** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @sesheta` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=thoth-station%2Fstorages).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/thoth-station/storages/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["sesheta"]} -->

sesheta commented:
New changes are detected. LGTM label has been removed.

VannTen commented:
Agree for some squashing. I kept 3 distinct commits though, to separate
docstrings modifications from refactors.

VannTen commented:
I had forgotten the `adviser` -> `storages` fix I don't know why.
This should be ready now :+1:

mayaCostantini reviewed (approved):
lgtm, thanks for the changes :100:

sesheta commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/thoth-station/storages/pull/2660#pullrequestreview-1054606113" title="LGTM">harshad16</a>*, *<a href="https://github.com/thoth-station/storages/pull/2660#pullrequestreview-1066225822" title="Approved">mayaCostantini</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=thoth-station%2Fstorages).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/thoth-station/storages/blob/master/OWNERS)~~ [mayaCostantini]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->
